images/scripts/update-cilium-runtime-image.sh

```diff
@@ -16,6 +16,7 @@ cd "${root_dir}"
 
 # Retrieve image from parameter and remove tag if one was provided, let make-image-tag in charge of creating the tag
 image=${1}
+image_dir=${2}
 image="${image%%:*}"
 
 image_tag="$(WITHOUT_SUFFIX=1 "${script_dir}/make-image-tag.sh" images/runtime)"
```

Comment:
@Artyop did you mean to also update this reference?

```suggestion
image_tag="$(WITHOUT_SUFFIX=1 ${script_dir}/make-image-tag.sh ${image_dir})"
```

Reply:
@Artyop :wave: friendly ping

Reply:
I completely missed this notification last month, sorry about that. Yes this was supposed to be updated, let me do a quick fix.

Reply:
https://github.com/cilium/cilium/pull/39722